code/controllers/subsystem/air.dm

```diff
@@ -68,6 +71,19 @@ SUBSYSTEM_DEF(air)
 /datum/controller/subsystem/air/fire(resumed = 0)
 	var/timer = TICK_USAGE_REAL
 
+	if(currentpart == SSAIR_REBUILD_PIPENETS)
+		var/list/pipenet_rebuilds = pipenets_needing_rebuilt
+		for(var/thing in pipenet_rebuilds)
+			if(thing)
+				var/obj/machinery/atmospherics/AT = thing
+				AT.build_network()
```

Comment:
Why `if(thing)` ?
Do you expect null entries to be there?

Reply:
if something gets qdelled it needs to be checked for otherwise it runtimes

Reply:
fixed